plugin/manager.go

```diff
@@ -0,0 +1,124 @@
+package plugin
+
+import (
+	"fmt"
+	"github.com/cloudquery/cloudquery/logging"
+	"github.com/hashicorp/go-plugin"
+	"github.com/rs/zerolog/log"
+	"os"
+	"os/exec"
+	"sync"
+)
+
+var (
+	doOnce   sync.Once
+	instance *Manager
+)
+
+type Manager struct {
+	activeClients map[string]*plugin.Client
+	registry      map[string]CQProvider
+
+	lock sync.RWMutex
+}
+
+func (p *Manager) Setup() error {
+	return nil
+}
+
+func (p *Manager) Shutdown() {
+	p.lock.Lock()
+	defer p.lock.Unlock()
+	for n, c := range p.activeClients {
+		c.Kill()
+		delete(p.activeClients, n)
```

Comment:
I guess you can `p.activeClient = make()` just like you do with `p.registry`. why different initialisation?